pkg/domain/ipam.go

```diff
@@ -0,0 +1,31 @@
+package domain
+
+// Device represents a physical device with a network interface on the stored IP address.
+type Device struct {
+	ID     string
+	IP     string
+	Subnet string
```

Comment:
question: similar to my `Subnet` question - this should be the `SubnetID`, right?

Reply:
Yes, and I also just realized that we have subnet and device ID in the API response payload, but not customer ID. Think we should add it? I ask because that affects the client implementation work you just did.

Reply:
Ah good catch, yes I think we should add it here :+1:

Reply:
ok, thanks!